src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -172,18 +173,27 @@ Amplify.configure({
 ```
 
 ### Recap
-At this point your client is now configured to point to your AppSync API endpoint and you updated your authentication details based on the auth types you are using.
+At this point, your client is now configured to point to your AppSync API endpoint and you updated your authentication details based on the auth types you are using.
 
 [comment]: # (ADD TO RECAP Tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
 
 ## Connect your application code to the data backend
 
-In this job, you will connect your application code to the data stack by connecting and configuring the Amplify libraries. We will review how to install and configure Amplify libraries to help you complete this task.
+In this job, you will connect your application code to the data stack by connecting and configuring the Amplify Libraries. We will review how to install and configure Amplify Libraries to help you complete this task.
 > **Before you begin, you will need:**
-> - npm installed ([set up npm](https://docs.npmjs.com/getting-started))
-> - Data stack deployed
+> - npm installed. For guidance on setting up npm the first time, see "([Getting started](https://docs.npmjs.com/getting-started) on the npm Docs site.)
+> - A data stack deployed
 
-### Install and configure the Amplify libraries
+### Amplify codegen to generate queries, mutations, and subscriptions 
+
+Using the `amplify add codegen` command, you can add an AppSync API that is created using the AWS console. If your API is in a different Region than your current Region, the command asks you to choose the Region. If you are adding codegen outside of an initialized Amplify project, provide your introspection schema named `schema.json` in the same directory that you make the add codegen call from. **Note:** If you use the --apiId flag to add an externally created AppSync API, such as one created in the AWS console, you will not be able to manage this API from the Amplify CLI with commands such as `amplify api update` when performing schema updates. You cannot add an external AppSync API when outside of an initialized project.
+
+```bash
+amplify add codegen --apiId <apiId>
+```
+Executing this command will generate the queries, mutations, and subscriptions within the src/graphql folder.
```

Comment:
Same here with this sentence - can we expand on this - why am I doing this step here vs later with the next section? Can add detail on why it is needed in this order?